ceph/rbd/workflows/parallel.py

```diff
@@ -0,0 +1,39 @@
+import importlib
+
```

Comment:
module name `parallel` confuse with `ceph parallel` can we make it `parallel_exec.py`

Reply:
changed.